pom.xml

```diff
@@ -393,6 +395,7 @@
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_2.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3ORCA.cmake</exclude>
+              <exclude>src/ports/greenplum/cmake/FindGreenplum_5.cmake</exclude>
```

Comment:
Hmm. It looks still not having a license header in the master branch.
https://github.com/apache/incubator-madlib/blob/master/src/ports/greenplum/cmake/FindGreenplum_5.cmake
Are we going to add it to this file?